skysat/ndvi/script.js

```diff
@@ -1,22 +1,22 @@
 //VERSION=3
 function setup() {
   return {
-    input: [{"bands": ["Red", "NIR","dataMask"]}],
-    output: { bands: 4}
+    input: [{ "bands": ["Red", "NIR", "dataMask"] }],
+    output: { bands: 4 }
   }
 }
 var f = 10000
 function evaluatePixel(sample) {
-  var NDVI = index (sample.NIR/f, sample.Red/f)
-  return valueInterpolate (NDVI,
-    [-1, -0.5, 0.2, 0.3, 0.4, 0.5, 1.0],
+  var NDVI = index(sample.NIR / f, sample.Red / f)
```

Comment:
An index will be the same if you apply the factor or not. You can remove the `f` here:

```
var NDVI = index (sample.NIR, sample.Red)
```